lib/routes/withgoogle/explorables.ts

```diff
@@ -0,0 +1,56 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    name: 'PAIR - AI Exploreables',
+    url: 'pair.withgoogle.com/explorables',
+    path: '/explorables',
+    maintainers: ['cesar'],
+    example: '/withgoogle/explorables',
+    radar: [
+        {
+            source: ['pair.withgoogle.com/explorables'],
+            target: '',
+        },
+    ],
+    handler: async () => {
+        const baseUrl = 'https://pair.withgoogle.com';
+        const response = await ofetch(baseUrl + '/explorables', {
+            method: 'GET',
+        });
+        const $ = load(response);
+        const items = await Promise.all(
+            $('div.explorable-card')
+                .map(async (_, el) => {
+                    const title = $(el).find('h3').text();
+                    const image = $(el).find('img').attr('src');
+                    const link = $(el).find('a').attr('href');
+                    let description;
+                    if (link) {
+                        description = await cache.tryGet(link, async () => {
```

Comment:
Please cache the whole returned object
https://github.com/DIYgod/RSSHub/blob/134451c53b2e8ea9c8d4528abcee7ba710686143/lib/routes/withgoogle/explorables.ts#L41-L46